core/kazoo/src/kz_rel.erl

```diff
@@ -0,0 +1,6 @@
+-module(kz_rel).
+-export([start/0]).
+
+start() ->
+	{ok, Cwd} = file:get_cwd(),
+	code:add_path(Cwd).
```

Comment:
Could you instead update dev-start-apps.sh? This feels like something that could be done with just L8 from said file